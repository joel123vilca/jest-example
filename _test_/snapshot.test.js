import { getCharacter } from '../snapshot';
import Rick from '../rick';

describe('Es hora de la intentadas', () => {
    test(Snapshot, () => {
        expect(getCharacter(rick)).toMatchSnaps();
    });
});