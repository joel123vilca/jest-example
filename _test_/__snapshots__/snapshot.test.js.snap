// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Es hora de las intantaneas Siempre fallara la instantanea 1`] = `
Object {
  "createAt": 2019-01-05T04:06:45.247Z,
  "id": 17,
}
`;

exports[`Es hora de las intantaneas Snapshot 1`] = `
Object {
  "id": 1,
  "name": "Rick Sanches",
  "species": "Human",
  "status": "Alive",
}
`;

exports[`Es hora de las intantaneas Tenemos una excepsion dentro del codigo 1`] = `
Object {
  "id": Any<Number>,
  "name": "Oscar Barajas Tavares",
}
`;
